.github/workflows/ci.yml

```diff
@@ -141,11 +141,13 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.8
-      - name: Install bazel
-        run: ./ci/run_ci.sh install_bazel
+      - name: Install Bazel
+        run: |
+          pip install psutil
+          pip install pyarrow==14.0.0
```

Comment:
Do we need to install pyarrow in py script?

Reply:
We can do this by moving install into py and then calling the shell through py to execute install.

Reply:
Sounds good to me. Can we install `psutil` and `pyarrow` in py script in this PR? The import for `psutil` can be moved into function to avoid import error

Reply:
Done.